### PR TITLE
Uv Rotation changed over to SimpleFloatSpinner

### PR DESCRIPTION
**Pull request by freyinds**

### CODE REVIEW

File: editor/CHANGES [hunk 1]
```diff
@@ -1,19 +1,12 @@
 [0.6.0] ~
-- [BREAKING CHANGE] Migrated from Kryo to JSON for .registry and .pro files. Migration handled automatically.
```
Comment (Dgzt) [inline editor/CHANGES:2]: Why did you remove these lines?

Comment (freyinds) [reply editor/CHANGES:2]: I apologize. Don't know how I managed to do that.

Comment (JamesTKhan) [reply editor/CHANGES:2]: Hm I know you pushed a commit to bring that line back, but its not just that line, other lines also removed click on "files changed" in on this PR and you will see other lines in the Changes file removed that should not be.

**Comment by JamesTKhan** [comment]
Looks good on my end, thanks!
